packages/shared/src/graphql/fragments.ts

```diff
@@ -55,7 +55,7 @@ export const SOURCE_BASE_FRAGMENT = gql`
     membersCount
     privilegedMembers {
       user {
-        ...UserShortInfo
+        id
```

Comment:
Fixes Feed request (got broken due to some duplicated importing of user info fragment) - originally just an id from production, so nothing will break as this is basically reverting.